packages/api/core/src/api/package.ts

```diff
@@ -245,6 +247,7 @@ export const listrPackage = ({
               signalPackageDone.get(getTargetKey({ platform: pPlatform, arch: pArch }))?.pop()?.();
               done();
             },
+            ...resolveHooks(forgeConfig.packagerConfig.afterComplete, ctx.dir),
```

Comment:
I talked to @MarshallOfSound and he mentioned that all the hooks aside from the `afterComplete` one should already be exposed via Packager 🤔

Reply:
All of the hooks added in https://github.com/electron/electron-packager/pull/1297 (including `afterComplete`) are not currently exposed, so I believe he was talking about the older hooks since that PR only landed a few months ago